lib/__test__/print.test.js

```diff
@@ -36,8 +36,10 @@ describe('print', () => {
     describe('fractions', () => {
         const tests = [
             ['(x + 1) / 1', '(x + 1) / 1'],
-            ['1/2/3', '1 / 2 / 3'], // (1/2) / 3
-            ['1*2/3', '1 * 2 / 3'], // 1 * (2/3)
+            ['2/3 + 5', '2/3 + 5'],
+            ['2/x + 5', '2 / x + 5'],
```

Comment:
wanna do it for all fractions where numerator and denominator are both either identifier or number?